appveyor.yml

```diff
@@ -0,0 +1,45 @@
+version: 1.0.{build}
+
+clone_folder: c:\projects\osqp-eigen
+
+environment:
+  Eigen3_DIR:          C:/Program Files (x86)/Eigen/lib/cmake/eigen3/
+
+os:
+ - Visual Studio 2015
+ - Visual Studio 2017
+
+install:
+  # Check env variables
+  - cmd: echo CMAKE_PREFIX_PATH %CMAKE_PREFIX_PATH%
+  - cmd: echo PATH %PATH%
+build:
+
+build_script:
+  # download and build osqp
+  - cd c:\projects
+  - git clone --recursive https://github.com/oxfordcontrol/osqp.git
```

Comment:
You can safely use `--depth 1` in order to clone only the tree of the last commit

Reply:
Done in dbda925